ATMS-597-SP-2020-Project-1/modules/tempconvert.py

```diff
@@ -0,0 +1,182 @@
+"""Temperature convertor between Celcius, Fahrenheit and Kelvin"""
+# This code is part of a class assignment for ATMS 597, Spring 2020,
+# at the University of Illinois at Urbana Champaign.
+# Use this class function to convert temperature data from already
+# known units to different units.
+# It supports conversion to Celcius, Fahrenheit and Kelvin.
+import numpy as np
+
+class tempconvert:
+    """Temperature convertor for numpy arrays and lists"""
+
+    # mem attribute will keep memory of the input data type
+    # Initialized as 0, it will become 1 if list is input
+    mem = 0
+
+    def __init__(self,in_temp):
+        """Create a temperature object.
+        The input is assigned to the object as class instance. Method calls 
+        are used to convert to desired units and retrieve output.
+
+        **Arguments:** 
+
+        *in_temp*
+            Input temperature. This can be numpy ndarray, or a list. Missing
+            values are not permitted. The units of temperature must be known,
+            and appropriate method be used to convert to desired units.
+          
+        **Returns:**
+
+        *temp*
+            A temperature convertor instance
+
+        **Example:**
+        
+        If in_temp is a numpy ndarray or a list:
+            import tempconvert
```

Comment:
`import tempconvert` needs to be replace by `from tempconvert import tempconvert` at 7 locations